src/test/java/com/epam/tc/hw2/TestExercise1HomePage.java

```diff
@@ -0,0 +1,99 @@
+        webDriver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+        softAssertions = new SoftAssertions();
+    }
+
+    @Test
+    void checkExercise1Scenario() {
+        // 1. Open test site by URL
+        webDriver.navigate().to("https://jdi-testing.github.io/jdi-light/index.html");
+        // 2. Assert Browser title
+        softAssertions.assertThat(webDriver.getTitle()).isEqualTo(expectedPageTitle);
+        // 3. Perform login
+        webDriver.findElement(By.id("user-icon")).click();
+        webDriver.findElement(By.id("name")).sendKeys("Roman");
+        webDriver.findElement(By.id("password")).sendKeys("Jdi1234");
+        webDriver.findElement(By.id("login-button")).click();
+        // 4. Assert Username is loggined
+        WebElement username = webDriver.findElement(By.id("user-name"));
+        softAssertions.assertThat(username.isDisplayed()).isTrue();
+        softAssertions.assertThat(username.getText()).isEqualTo(expectedUserName);
+        // 5. Assert that there are 4 items on the header section are displayed and they have proper texts
+        List<WebElement> navigationBarItems = webDriver.findElements(
+            By.cssSelector("ul[class=\"uui-navigation nav navbar-nav m-l8\"]>li"));
+        softAssertions.assertThat(navigationBarItems.size()).isEqualTo(expectedNavigatorBarItems.size());
+        assertThatAllElementsExpectedlyDisplayed(navigationBarItems, expectedNavigatorBarItems);
+        // 6. Assert that there are 4 images on the Index Page and they are displayed
+        List<WebElement> benefitIcons = webDriver.findElements(By.cssSelector("span[class^=\"icons-benefit\"]"));
+        softAssertions.assertThat(benefitIcons.size()).isEqualTo(expectedIconsNumber);
+        // 7. Assert that there are 4 texts on the Index Page under icons and they have proper text
+        List<WebElement> benefitTexts = webDriver.findElements(By.cssSelector("span[class=\"benefit-txt\"]"));
+        assertThatAllElementsExpectedlyDisplayed(benefitTexts, expectedBenefitTexts);
+        // 8. Assert that there is the iframe with “Frame Button” exist
+        WebElement buttonFrame = webDriver.findElement(By.id("frame"));
+        softAssertions.assertThat(buttonFrame.isDisplayed()).isTrue();
+        // 9. Switch to the iframe and check that there is “Frame Button” in the iframe
+        webDriver.switchTo().frame(buttonFrame);
+        WebElement inFrameButton = webDriver.findElement(By.id("frame-button"));
+        softAssertions.assertThat(inFrameButton.isDisplayed()).isTrue();
+        // 10. Switch to original window back
+        webDriver.switchTo().parentFrame();
+        // 11. Assert that there are 5 items in the Left Section are displayed and they have proper text
+        List<WebElement> sidebarElements = webDriver.findElements(By.cssSelector("ul[class=\"sidebar-menu left\"]>li"));
+        softAssertions.assertThat(sidebarElements.size()).isEqualTo(expectedSidebarElements.size());
```

Comment:
You probably forgot to softAssertions.assertAll() to check all assertion

Reply:
Yes, thank you!